src/main/scala/tech/cryptonomic/conseil/tezos/FeeOperations.scala

```diff
@@ -3,19 +3,27 @@ package tech.cryptonomic.conseil.tezos
 import com.typesafe.config.ConfigFactory
 import com.typesafe.scalalogging.LazyLogging
 import tech.cryptonomic.conseil.Lorre.db
-import scala.concurrent.ExecutionContext.Implicits.global
+import tech.cryptonomic.conseil.tezos.{TezosDatabaseOperations => Tdb}
 
-import scala.concurrent.Await
-import scala.concurrent.duration.{Duration, SECONDS}
+import scala.concurrent.{Future, ExecutionContext}
 import scala.util.{Failure, Success, Try}
+import slick.dbio.DBIOAction
 
 /**
   * Helper classes and functions used for average fee calculations.
   */
 object FeeOperations extends LazyLogging {
 
-  private val conf = ConfigFactory.load
-  private val awaitTimeInSeconds = conf.getInt("dbAwaitTimeInSeconds")
+  private val operationKinds = List(
```

Comment:
If your filter in Database Operations is semantically correct, this list might be redundant.

Reply:
I thought it's needed, it's used on line 51 to sequence the fee calculations for each operation kind.

I just moved the definition here so it stands out better both for reference and for future changes

Reply:
You're right! Redacted